xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/actionbuttons/actionButtons.js

```diff
@@ -425,7 +399,13 @@ var XWiki = (function(XWiki) {
     },
     // Reload the editors in case the user want to loose his change.
     reloadEditor : function () {
-      window.location.reload();
+      // We don't rely on window.location.reload() since it might keep cached data from the form
+      // we don't rely on window.location.reload(true) either since it's unclear if it's properly supported by
+      // all browsers. Instead we rely on a query parameter with the current date.
+      // URLSearchParams is not supported by IE6 but a polyfill must be available
+      var params = new URLSearchParams(window.location.search);
```

Comment:
@mflorea I'd need your input on this one about the polyfill.

Reply:
So after discussing it on #xwiki it appears that I shouldn't use this API since it's not documented on https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/js/polyfills.txt